python/FlexSource.py

```diff
@@ -53,25 +54,48 @@ def center_freq(self):
         return self._center_freq
 
     def set_center_freq(self,center_freq):
-        """ 
+        """
         Sets the center frequency of the underlying Panadapter
-        
+
         Args:
             center_freq: the new center frequency
         """
-        self._center_freq = center_freq
+        self._center_freq = center_freq/1000000
```

Comment:
I think it might be good to add an internal method on this class for doing the conversion.  The only reason is it limits the number of "magic" numbers in the class and at least all of the logic for that is in one spot.